examples/sqlite_remember.rs

```diff
@@ -0,0 +1,94 @@
+use teloxide::{
```

Comment:
Is this any different from `redis_remember`? Can we merge the two?

Reply:
https://github.com/teloxide/teloxide/issues/504